js/data.js

```diff
@@ -1,8 +1,35 @@
 /* exported data */
-
+var $form = document.querySelector('#entry-form');
+var $title = document.querySelector('.title');
+var $photoUrl = document.querySelector('.photo-url');
+var $notes = document.querySelector('.notes');
```

Comment:
All of this should be in `main.js`. The only code that should be in `data.js` should be your data model and any code related to localStorage